### PR TITLE
Added readonly status for the cluster.

### PR DESCRIPTION
**Pull request by NelyDavtyan**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing applications:)

**What changes did you make?** (Give an overview)
Issues #1185 
Screen Shot 2022-01-17 at 4 08 26 PM

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "X" next to an item)

- [x] No need to
- [x] Manually(please, describe, when necessary)
- [x] Unit checks
- [x] Integration checks
- [x] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings(e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

**Comment by workshur** [comment]
@NelyDavtyan Could you add screenshot of your change to description of current PR?

File: kafka-ui-react-app/src/components/Dashboard/ClustersWidget/ClustersWidget.tsx [hunk 1]
```diff
@@ -86,7 +86,14 @@ const ClustersWidget: React.FC<Props> = ({
           <tbody>
             {chunkItem.data.map((cluster) => (
               <tr key={cluster.name}>
-                <td>{cluster.name}</td>
+                <td>
+                  {cluster.readOnly && (
+                    <div className="tag has-margin-right is-info is-light">
```
Comment (Hurenka) [inline kafka-ui-react-app/src/components/Dashboard/ClustersWidget/ClustersWidget.tsx:91]: We're trying to use own styles with the styled-components and without classnames, aren't we?

**Review by Hurenka — CHANGES_REQUESTED** [review]

File: kafka-ui-react-app/src/theme/theme.ts [hunk 2]
```diff
@@ -34,6 +34,12 @@ export const Colors = {
   yellow: {
     '10': '#FFEECC',
   },
+  blue: {
+    '10': '#e3f2fd',
+    '20': '#bbdefb',
+    '30': '#90caf9',
+    '40': '#64b5f6',
```
Comment (workshur) [inline kafka-ui-react-app/src/theme/theme.ts:41]: Are these colors from figma?

**Review by workshur — APPROVED** [review]

**Review by Hurenka — APPROVED** [review]

File: kafka-ui-react-app/src/components/common/Tag/Tag.styled.tsx [hunk 3]
```diff
@@ -1,7 +1,8 @@
 import styled from 'styled-components';
 
 interface Props {
-  color: 'green' | 'gray' | 'yellow' | 'red' | 'white';
+  className?: string;
```
Comment (Hurenka) [inline kafka-ui-react-app/src/components/common/Tag/Tag.styled.tsx:4]: remove unused className prop, please

**Review by workshur — APPROVED** [review]

**Review by Motherships — APPROVED** [review]

**Pull request merged**